proof-new/bv-concat-merge.smt2

```diff
@@ -0,0 +1,11 @@
+(set-info :smt-lib-version 2.6)
```

Comment:
These should be moved under directory `test/regress/cli/regress0/proof`.

Reply:
This folder is only for files that demonstrate the new rewrite rules can work. The output may change in the next few iterations so I'm putting them here. Eventually the entire `proof-new` folder will be deleted. I planned to delete that folder before this PR.

Reply:
I think we should go ahead and move the files to the proper place on this PR.

Reply:
Is this for regression testing? I read a few files from `test/regress/cli/regress0/proof` and I didn't find an accompanying anticipated output file corresponding to the `.smt2` case files.

Reply:
We do not keep separate expected output files.  Instead expected output is either determined simply by `(set-info :status X)` or otherwise given in comments `; EXPECT:`.

See for example: https://github.com/cvc5/cvc5/blob/main/test/regress/cli/regress0/bug421.smt2#L2

Reply:
The purpose of these files is to test whether cvc5 is exercising the bitvector rewrite rules in the rewrite steps, so I think if we just expect unsat it defeats the purpose of these tests. The tests are written in an way that is obviously unsat as well.

Reply:
We often add regressions for the purpose of increasing coverage, regardless of the expected output. Notice that you can specify command line parameters:
`; COMMAND-LINE: --proof-granularity=dsl-rewrite`

Reply:
done